pkg/adaptor/proxy/service.go

```diff
@@ -309,17 +327,47 @@ func (s *proxyService) GetMetrics(ctx context.Context, req *pb.GetMetricsRequest
 
 func (s *proxyService) CreateSandbox(ctx context.Context, req *pb.CreateSandboxRequest) (res *types.Empty, err error) {
 
+	logger.Printf("CreateSandbox: hostname:%s sandboxId:%s", req.Hostname, req.SandboxId)
+	if len(req.Dns) > 0 {
+		logger.Print("    dns:")
+		for _, d := range req.Dns {
+			logger.Printf("         %s", d)
+		}
+
+		logger.Print("    Eliminated the DNS setting above from CreateSandboxRequest to stop updating /etc/resolv.conf on the peer pod VM")
+		logger.Print("    See https://github.com/confidential-containers/cloud-api-adaptor/issues/98 for the details.")
+		req.Dns = nil
+	}
+	if len(req.Storages) > 0 {
+		logger.Print("    storages:")
+		for _, s := range req.Storages {
+			logger.Printf("         mountpoint:%s source:%s fstype:%s driver:%s", s.MountPoint, s.Source, s.Fstype, s.Driver)
+		}
+	}
```

Comment:
nit: we should make these printfs part of DEBUG log

Reply:
I agree. We are using the logger in the standard Go library. It only supports print, fatal, and panic. We need to switch to a modern logging library.